src/language-js/printer-estree-json.js

```diff
@@ -52,6 +52,8 @@ function genericPrint(path, options, print) {
       return JSON.stringify(node.value);
     case "Identifier":
       return JSON.stringify(node.name);
+    case "V8IntrinsicIdentifier":
+      return JSON.stringify(`%${node.name}`);
```

Comment:
Can there be intrinsics in json files?

Reply:
No. I guess I misunderstood what this file does, I thought it's printing out arbitrary ESTree ASTs as JSON.